doc/user/error-reports.rst

```diff
@@ -117,33 +117,56 @@ wide, but a 16 pixels version is used if the fringe is `wide enough
    distribution please take a look at its documentation if you're unsure about
    the appearance of Flycheck's indicators.
 
-   Note that we discourage you from changing the shape of Flycheck’s fringe
-   indicators.
-
 You can customise the location of these indicators (left or right fringe) with
-`flycheck-indication-mode` which also lets you turn off these indicators
-completely:
+`flycheck-indication-mode`, which also lets you turn off these indicators
+completely; additionally, you can move these indicators into the margins instead
+of the fringes:
 
 .. defcustom:: flycheck-indication-mode
 
    How Flycheck indicates errors and warnings in the buffer fringes:
 
    ``left-fringe`` or ``right-fringe``
-      Use the left or right fringe respectively.
+      Use the left or right fringe respectively.  Fringes can only contain
+      monochrome bitmaps, so Flycheck draws small pixel-art arrows.
+
+   ``left-margin`` or ``right-margin``
+      Use the left or right margin respectively.  Margins can support all of
+      Emacs' rendering facilities, so Flycheck uses the ``»`` character, which
+      scales with the font size.
 
    ``nil``
-      Do not indicate errors and warnings in the fringe.
+      Do not indicate errors and warnings in the fringe or in the margin.
+
+By default, Emacs displays fringes, but not margins.  With ``left-margin`` and
+``right-margin`` indication modes, you will need to enable margins in your
+``.emacs``.  For example:
+
+.. code-block:: elisp
 
-The following faces control the colours of the fringe indicators.  However they
-do not let you change the shape of the indicators—to achieve this you'd have to
-redefine the error levels with `flycheck-define-error-level`.
+   (setq-default left-fringe 1 right-fringe 8
```

Comment:
```suggestion
   (setq-default left-fringe-width 1 right-fringe-width 8
```

Reply:
Good catch, thanks a lot! I squashed this into a previous commit.